### ISSUE TITLE
PHPUnit logging configuration is not respected when passed to `php artisan test`

### ISSUE THREAD

**Issue by lucavallin** [laravel/framework#33986]
- Laravel Version: 7.10.3
- PHP Version: 7.4.8

### Description:

PHPUnit logging configuration is not respected when passed to `php artisan test`.

### Steps To Reproduce:

In my `phpunit.xml` I have:

```
<logging>
        <log type="coverage-text" target="php://stdout" showUncoveredFiles="false" showOnlySummary="true" />
    </logging>
```

If I run `phpunit`, at the end I get a coverage report in the shape of:

```
Code Coverage Report Summary:
  Classes: 50.00% (6/12)     
  Methods: 63.64% (14/22)    
  Lines:   58.82% (30/51)    
```

If I run `php artisan test` instead, I only get:

```
Tests:  7 passed
 Time:   3.04s
```

However, if I run `php artisan test --coverage-text=coverage.txt`, a `coverage.txt` with the information I am expecting is created, but I really need this report in my `stdout` and I am not able to do that.

**Comment by driesvints** [comment]
please open up an issue nunomaduro/collision which provides this command.

**Comment by markjaquith** [comment]
It seems to have been fixed in version `4.2.0`.

`composer require --dev "nunomaduro/collision ^4.2"`
